ts/parsers/elementparser.ts

```diff
@@ -7,49 +7,134 @@ import * as isEmpty from "lodash.isempty";
 /**
  * Entry point for all Parsers
  */
+const SUPPORTED_PLACEHOLDERS = ["body", "ctrTitle", "pic", "subTitle", "tbl", "title"];
 class PowerpointElementParser {
     private element;
 
-    public getProcessedElement(rawElement, slideRelationships): PowerpointElement {
+    public isNonSupportedPlaceholder() {
+        if (checkPath(this.element, '["p:nvSpPr"][0]["p:nvPr"][0]["p:ph"][0]["$"]["type"]')) {
+            const type = getValueAtPath(
+                this.element,
+                '["p:nvSpPr"][0]["p:nvPr"][0]["p:ph"][0]["$"]["type"]'
+            );
+            if (!SUPPORTED_PLACEHOLDERS.includes(type)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+    public isPlaceholderListElement(slideLayoutTables): boolean {
+        if (checkPath(this.element, '["p:nvSpPr"][0]["p:nvPr"][0]["p:ph"][0]["$"]["idx"]')) {
+            const placeholderIdx = getValueAtPath(
+                this.element,
+                '["p:nvSpPr"][0]["p:nvPr"][0]["p:ph"][0]["$"]["idx"]'
+            );
+            let slideLayoutSpNode = undefined;
+            slideLayoutSpNode = slideLayoutTables["idxTable"][placeholderIdx];
```

Comment:
`slideLayoutSpNode` isn't reassigned so these lines can be combined and `slideLayoutSpNode` changed to `const`

Reply:
done